vioserial/sys/Control.c

```diff
@@ -49,7 +49,8 @@ VIOSerialSendCtrlMsg(
         virtqueue_kick(vq);
         while(!virtqueue_get_buf(vq, &len))
         {
-           KeStallExecutionProcessor(50);
+            LARGE_INTEGER interval = {0};
```

Comment:
It looks like passing 0 for interval is not really documented as it's neither positive nor negative. Would it be maybe cleaner to use -1?

Reply:
I agree. Zero interval just makes KeDelayExecutionThread to call NtYieldExecution with minimal additional actions, but it not obvious, and not documented.